lighthouse-core/gather/gatherers/dobetterweb/response-compression.js

```diff
@@ -58,7 +58,11 @@ class ResponseCompression extends Gatherer {
 
     const driver = options.driver;
     return Promise.all(textRecords.map(record => {
-      return driver.getRequestContent(record.requestId).then(content => {
+      const contentPromise = driver.getRequestContent(record.requestId);
+      const timeoutPromise = new Promise((resolve, reject)=>{
```

Comment:
from patrick: 

> we should util a promise race timeout like this and do it everywhere

> this brings up a good point, should we race all gatherer afterPasses at some max timeout?

Reply:
To make it a bit more consistent, make all the timeout to be 3000 for now.

Reply:
ok cool 👍  

can we one-liner these timeouts? `new Promise(resolve => setTimeout(resolve, 3000))`

Reply:
Done